src/app/app-routing.module.ts

```diff
@@ -1,9 +1,17 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { HomeComponent } from './core/home/home.component';
 import { LoginComponent } from './core/login/login.component';
+import { DashBoardComponent } from './features/dashboard/dashboard.component';
 
 const routes: Routes = [
   { path: "", component: LoginComponent },
+  { path: "login", component: LoginComponent },
+  {
+    path: "lms", component: HomeComponent, children: [
```

Comment:
Looks fine now, no for all new components you need to add childrens path here inside home component